.rubocop.yml

```diff
@@ -7,7 +7,7 @@ AllCops:
 
 # Baseline code complexity metrics - we should try and reduce these over time
 Metrics/AbcSize:
-  Max: 47
+  Max: 71.5
```

Comment:
The size is the problem. If we keep letting this number grow then it becomes fairly meaningless and it won't flag methods that are getting out of hand. Some methods are just going to be large and it won't make sense to carve them up into smaller pieces, but those should be the exceptions. 

By disabling this on case-by-case basis, when we see a disabled around a particular method in review it's going to catch our attention and we can make a judgement call at that point.

I can't say for sure what the optimum size for this should be, but the methods it's been flagging lately are clearly too big so I would venture that the current value is pretty close to the max we want.